.vscode/settings.json

```diff
@@ -15,5 +15,9 @@
     "MD025": {
       "front_matter_title": ""
     }
-  }
+  },
+  "cSpell.words": [
+    "Differentrequirement",
+    "Equalrequirement"
+  ]
```

Comment:
Could you remove this setting configurations?

Reply:
The requested deletion has been completed.